code/modules/suppressions/extraction.dm

```diff
@@ -465,3 +470,60 @@
 	pillar_storm_cooldown = world.time + pillar_storm_cooldown_time
 	charging = FALSE
 	icon_state = icon_living
+
+/mob/living/simple_animal/hostile/megafauna/arbiter/proc/SpawnSpikes()
+	spikes_cooldown = world.time + spikes_cooldown_time
+	for(var/mob/living/carbon/human/H in GLOB.player_list)
+		if(!H.client)
+			continue
+		if(H.stat)
+			continue
+		if(H.z != z)
+			continue
+		if(H.is_working)
+			continue
+		var/turf/T = get_turf(H)
+		new /obj/effect/arbiter_spike(T)
+		for(var/turf/open/TT in view(2, T))
+			if(prob(50))
+				continue
+			if(locate(/obj/effect/arbiter_spike) in TT)
+				continue
+			new /obj/effect/arbiter_spike(TT)
+
+/* Objects & Effects */
+// Spikes: Arbiter will spawn these from time to time around the agents, which after short delay will damage everything
+// on its tile for 100 BLACK damage
+/obj/effect/arbiter_spike
+	name = "dark spikes"
+	desc = "A weird construction. You should probably stay away from it..."
+	icon_state = "binah_spike"
+	alpha = 0
+	anchored = TRUE
+	density = FALSE
+	/// How much time must pass after Initialize() to activate
+	var/activation_delay = 2 SECONDS
+	/// Amount of BLACK damage done on activation
+	var/damage = 150
```

Comment:
Leaving a note that this may be a bit too much damage? Base Lobotomy says 33-40 damage, which should be ~82-100 damage for us. (We tend to do like, 2.5x from what I recall).